client/src/components/Replies/index.jsx

```diff
@@ -0,0 +1,44 @@
+import React, { Component } from 'react';
+import './style.css';
+import Conversation from '../Conversation';
+
+class Replies extends Component {
+    state = {
+      replies: [],
+    }
+
+    componentDidMount() {
+      this.setState({
+        replies: [
+          {
+            email: 'jamalat@getsMaxListeners.com',
+            message: 'Lorem ipsum dolor sit amet, consectetuer adipiscing elit. Aenean commodo ligula eget dolor. Aenean massa. Cum sociis natoque penatibus et magnis dis parturient montes, nascetur ridiculus mus. Donec quam felis, ultricies nec, ',
+          },
+          {
+            email: 'jamalat@getMadxListeners.com',
+            message: 'Lorem ipsum dolor sit amet, consectetuer adipiscing elit. Aenean commodo ligula eget dolor. Aenean massa. Cum sociis natoque penatibus et magnis dis parturient montes, nascetur ridiculus mus. Donec quam felis, ultricies nec, ',
+          },
+          {
+            email: 'jamalat@getMaxListeners.com',
+            message: 'Lorem ipsum dolor sit amet, consectetuer adipiscing elit. Aenean commodo ligula eget dolor. Aenean massa. Cum sociis natoque penatibus et magnis dis parturient montes, nascetur ridiculus mus. Donec quam felis, ultricies nec, ',
+          },
+        ],
+      });
+    }
+
+    render() {
+      return (
+        <React.Fragment>
```

Comment:
unnecessary fragment. component is already wrapped in the ```replies-component``` if I didn't make a scoping mistake.

we should also use destructuring syntax on fragment just like we do on component.